requirements/codebases.txt

```diff
@@ -1,5 +1,6 @@
 mmcls>=1.0.0rc2
 mmdet @ git+https://github.com/open-mmlab/mmdetection.git@dev-3.x
+mmdet3d==1.1.0rc1
```

Comment:
this version has bug of installing mmdet3d with mim
https://github.com/open-mmlab/mmdetection3d/blob/28fe73d25013bd880d22b5952da7cd16f5116826/requirements/mminstall.txt#L1

Reply:
done.